examples/cloudflare-redwood/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@redwoodjs/starter-drizzle",
+  "version": "1.0.0",
+  "description": "A RedwoodSDK starter for projects with a database using Drizzle",
+  "main": "index.js",
+  "type": "module",
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "build": "vite build",
+    "dev": "NODE_ENV=${NODE_ENV:-development} vite dev",
+    "dev:init": "rw-scripts dev-init",
+    "preview": "bun run build && bun run vite preview",
+    "worker:run": "rw-scripts worker-run",
+    "clean": "bun run clean:vite",
+    "clean:vite": "rm -rf ./node_modules/.vite",
+    "release": "rw-scripts ensure-deploy-env && bun run clean && prisma generate && RWSDK_DEPLOY=1 bun run build && wrangler deploy",
+    "format": "prettier --write ./src",
+    "migrate:new": "drizzle-kit generate",
+    "migrate:dev": "wrangler d1 migrations apply DB --local",
+    "seed": "bun run worker:run ./src/db/seed.ts",
+    "check": "bun run types",
+    "types": "bun run tsc"
+  },
+  "dependencies": {
+    "@redwoodjs/sdk": "0.0.51",
```

Comment:
not a big deal but my branch uses a newer version and already resolves the breaking changes since this version if you wanted to use that as reference

Reply:
Thanks for calling that out. I used the `digit` command that matches the docs guide too. I'll upgrade the example.